sass/patternfly-react/_patternfly-react.scss

```diff
@@ -1,3 +1,13 @@
 /**
   Patternfly React Partials
 */
+.label-strong {
+  font-weight: 600;
+}
+.display-inline-block {
+  display: inline-block;
+}
+.label-text {
+  font-size: 12px !important;
+  color: black;
+}
```

Comment:
Sorry, updates to the [CONTRIBUTING.MD](https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md) file had not yet been merged. It has been merged now. Please see the  [Adding Styling for your Components](https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md#adding-styling-for-your-components) section.

Also, these selectors should be qualified so as not to conflict with an application's selectors.